kubernetes/client/configuration.py

```diff
@@ -265,13 +265,17 @@ def debug(self, value):
         if self.__debug:
             # if debug status is True, turn on debug logging
             for _, logger in six.iteritems(self.logger):
+                if logger.level == logging.DEBUG:
+                    continue                
```

Comment:
This file is generated by Openapi Generator https://github.com/openapitools/openapi-generator. Please send the fix to the upstream